### PR TITLE
Cleanup Text in Markdown Files

### PR DESCRIPTION
**Pull request by BrayFlex**
- Enforced formatting consistency where overlooked
- Improved the language used
- Fixed some spelling mistakes
- Rewrote the project description

Its not much but this project is awesome and I wanted to contribute to the work done.

### CODE REVIEW

**Comment by Natizyskunk** [comment]
@BrayFlex,
Thanks a lot for your contribution.
Please correct my reviews this way I can merge your work 🙂

**Comment by BrayFlex** [comment]
> @BrayFlex, Thanks a lot for your contribution. Please correct my reviews this way I can merge your work 🙂

I don't think you're review went through

**Comment by Natizyskunk** [comment]
@BrayFlex,

Don't you see those reviews ?
image

**Comment by BrayFlex** [comment]
> @BrayFlex,
> 
> Don't you see those reviews ? image

Am I looking in the wrong play @Natizyskunk ? 
image

**Comment by Natizyskunk** [comment]
@BrayFlex,
If you can't so them in the "conversation" tab, you should be able to see them in the "Files changed" tab.
Please let me now if you can't see them.

**Review by Natizyskunk — COMMENTED** [review]
Please make the requested changes please 🙂

**Comment by BrayFlex** [comment]
> @BrayFlex, If you can't so them in the "conversation" tab, you should be able to see them in the "Files changed" tab. Please let me now if you can't see them.

@Natizyskunk Unfortunately there are no comments

**Comment by Natizyskunk** [comment]
> > @BrayFlex,
> > Don't you see those reviews ?
> 
> Am I looking in the wrong place @Natizyskunk ? image

@BrayFlex,
My bad I had forgotten to do something on my end to make it visible for you.
You should now be able to see them in the "conversation" tab. If not you should also be able to see them in the "Files changed" tab.

**Comment by BrayFlex** [comment]
> > > @BrayFlex,
> > > Don't you see those reviews ?
> > 
> > 
> > Am I looking in the wrong place @Natizyskunk ? image
> 
> @BrayFlex, My bad I had forgotten to do something on my end to make it visible for you. You should now be able to see them in the "conversation" tab. If not you should also be able to see them in the "Files changed" tab.

I see it now, will fix these up soon!

**Review by Natizyskunk — CHANGES_REQUESTED** [review]
Please make the requested changes please 🙂

**Comment by Natizyskunk** [comment]
> > > @BrayFlex,
> > > Don't you see those reviews ?
> > 
> > 
> > Am I looking in the wrong place @Natizyskunk ? image
> 
> @BrayFlex, My bad I had forgotten to do something on my end to make it visible for you. You should now be able to see them in the "conversation" tab. If not you should also be able to see them in the "Files changed" tab.

Perfect 😀

**Comment by Natizyskunk** [comment]
@BrayFlex,
Also keep in mind to always be up-to-date with the `develop` branch.

File: README.md [hunk 1]
```diff
@@ -54,16 +50,15 @@ To install it just follow thoses steps from within VSCode :
...
 then download your project, and from that point sync.  
 
 1. In `VS Code`, open a local directory you wish to sync to the remote server (or create an empty directory 
-that you wish to first download the contents of a remote server folder in order to edit locally). 
-2. `Ctrl+Shift+P` on Windows/Linux or `Cmd+Shift+P` on Mac open command palette, run `SFTP: config` command.
+that you wish to first download the contents of a remote server folder in order to edit locally).
+2. `Ctrl+Shift+P` on Windows/Linux or `Cmd+Shift+P` on Mac open command palette, run `SFTP: config` command
 3. A basic configuration file will appear named `sftp.json` under the `.vscode` directory.
-Edit the parameters to match your setup.
+Edit the configuration parameters to your remote server
```
[the remark this answers is not in the document]
Comment (BrayFlex) [reply README.md:60]: I wouldn't say so, you would not say: "Enter your matching login credentials", you'd just say "Enter your login credentials"

File: README.md [hunk 2]
```diff
@@ -77,22 +72,19 @@ For instance:
...
-4. `Ctrl+Shift+P` on Windows/Linux or `Cmd+Shift+P` on Mac open command palette.
-5. Type "sftp" and you'll now see a number of other comands.
-6. A good one to start with if you want to start with a remote folder is `SFTP: Download Project`.  This will download the directory shown in the `remotePath` setting in `sftp.json` to your local open directory. 
-7. Done - you can now edit locally and after each save it will upload to your remote file. 
-8. Enjoy !
+1. Save and close the `sftp.json` file.
+2. `Ctrl+Shift+P` on Windows/Linux or `Cmd+Shift+P` on Mac open command palette.
+3. Type `sftp` and you'll now see a number of other commands. You can also access many of the commands from the project's file explorer context menus.
+4. A good one to start with if you want to sync with a remote folder is `SFTP: Download Project`.  This will download the directory shown in the `remotePath` setting in `sftp.json` to your local open directory. 
+5. Done - you can now edit locally and after each save it will upload to sync your remote file with the local copy.
```
[the remark this answers is not in the document]
Comment (BrayFlex) [reply README.md:82]: Did you mean continue with 4?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Natizyskunk) [reply README.md]: @BrayFlex,
For me "Edit the configuration parameters with your remote server informations" would be a nice in-between just to make it clear for users about what they should do. Is it ok for you ?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Natizyskunk) [reply README.md]: @BrayFlex,
Yes, my bad.

should look like that : 
```md
1. In `VS Code`, open a local directory you wish to sync to the remote server (or create an empty directory .
2. `Ctrl+Shift+P` on Windows/Linux or `Cmd+Shift+P` on Mac open command palette, run `SFTP: config` command.
3. A basic configuration file will appear named `sftp.json` under the `.vscode` directory.
[...]
4. Save and close the `sftp.json` file.
5. `Ctrl+Shift+P` on Windows/Linux or `Cmd+Shift+P` on Mac open command palette.
[...]
```

File: README.md [hunk 3]
```diff
@@ -31,19 +27,19 @@ Very powerful, with smart features. Very simple, requires just three lines of co
...
-1. Select Extensions (Ctrl + Shift + X).
+To install just follow these steps from within VSCode:
+1. Select Extensions (Ctrl + Shift + X)
 2. Uninstall current sftp extension from @liximomo
-3. Open "More Action” menu(ellipsis on the top) and click “Install from VSIX…”.
+3. Open "More Action" menu(ellipsis on the top) and click "Install from VSIX…"
 4. Locate VSIX file and select.
 5. Reload VSCode.
-6. Voilà !
+6. Voilà!
```
Comment (Natizyskunk) [inline README.md:42]: @BrayFlex,
Please leave the punctuation at the end of each sentence.

File: README.md [hunk 4]
```diff
@@ -31,19 +27,19 @@ Very powerful, with smart features. Very simple, requires just three lines of co
 ## Installation
 
 ### Method 1 (Recommended : Auto update)
-1. Select Extensions (Ctrl + Shift + X).
+1. Select Extensions (Ctrl + Shift + X)
 2. Uninstall current sftp extension from @liximomo
 3. Install new extension directly from VS Code Marketplace : https://marketplace.visualstudio.com/items?itemName=Natizyskunk.sftp
-4. Voilà !
+4. Voilà!
```
Comment (Natizyskunk) [inline README.md:33]: @BrayFlex,
Please leave the punctuation at the end of each sentence.

**Review by Natizyskunk — CHANGES_REQUESTED** [review]
Just some minor changes and we should be good to go.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (BrayFlex) [reply README.md]: I think that works too

**Comment by BrayFlex** [comment]
@Natizyskunk I think that about fixes everything

**Review by Natizyskunk — APPROVED** [review]
Perfect ! 🙂

**Review by Natizyskunk — APPROVED** [review]

**Comment by Natizyskunk** [comment]
It's been merged in #244 😉

**Pull request merged**